promptsource/templates/piqa/templates.yaml

```diff
@@ -157,30 +227,33 @@ templates:
     - 'No'
     answer_choices_key: null
     id: f044def7-01c2-42de-b6ad-4e8c63ab2bf1
-    jinja: '{{goal}} {{sol1[0].lower() + sol1[1:]}}
+    jinja: 'Does this phrase make sense?
 
-      Does this phrase make sense?
+      {{goal}} {{sol1[0].lower() + sol1[1:]}}
+
+      Answer with {{answer_choices[0]}} or {{answer_choices[1]}}
 
       |||
 
       {{answer_choices[label]}}'
     metadata: !TemplateMetadata
       _do_eval: false
       _do_train: true
-      choices_in_prompt: false
-      metrics: []
-      original_task: null
+      choices_in_prompt: true
+      metrics:
+      - Accuracy
+      original_task: false
     name: Does this solution make sense? sol1
     reference: ''
   f42cd457-a14b-465a-a139-d7d2407a3bac: !Template
     answer_choices: null
     answer_choices_key: null
     id: f42cd457-a14b-465a-a139-d7d2407a3bac
-    jinja: 'Given a sentence, correct it if it doesn''t make sense.
+    jinja: 'Sentence: {{goal}} {{sol1[0].lower() + sol1[1:]}}
 
-      Input: {{goal}} {{sol1[0].lower() + sol1[1:]}}
+      If the sentence does not make sense, correct it so that it does make sense.
 
-      Output:
+      Answer:
```

Comment:
Need something in the template to instruct the model to re-copy if the solution makes sense